tests/test_gwas_py.py

```diff
@@ -0,0 +1,106 @@
+# encoding: utf-8
+
+"""
+Test module for ``gwas.py`` script
+"""
+
+import os
+import subprocess
+import tempfile
+
+p3_sif = os.path.abspath(os.path.join("singularity", "python3.sif"))
+cwd = os.getcwd()
+ref = os.path.join(cwd, "reference", "examples", "regenie")
+
+os.environ["SINGULARITY_BIND"] = (
+    f"{os.path.join(cwd,'reference')}:/REF:ro,"
+    f" {os.path.join(cwd, '..', 'reference')}:/REF2:ro"
```

Comment:
Best to drop `` f" {os.path.join(cwd, '..', 'reference')}:/REF2:ro"`` line, and avoid taking dependency on github.com/comorment/reference in unit-tests